nf_core/modules/info.py

```diff
@@ -163,7 +163,9 @@ def generate_module_info_help(self):
         elif self.remote_location:
             intro_text.append(
                 Text.from_markup(
-                    f":globe_with_meridians: Repository: [link=https://github.com/{self.remote_location}]{self.remote_location}[/]\n"
+                    ":globe_with_meridians: Repository: [link="
+                    f"{'' if self.remote_location.startswith('http') or self.remote_location.startswith('ssh') else 'https://github.com/'}"
+                    f"{self.remote_location}]{self.remote_location}[/link]\n"
```

Comment:
I think it only makes sense to create a clickable link if the remote location is a http URL. There is no need to keep the `https://github.com` prefix either since the remote location will always be a full URL.

Reply:
Oh sorry, I was testing with `nf-core/modules-test` which works without specifying the full url and got confused. You are right. This should also work with any kind of URL other than http

 ```suggestion
                    ":globe_with_meridians: Repository: "
                    f"{ "[link={self.remote_location}]" if self.remote_location.startswith('http') else ''}"
                    f"{self.remote_location}"
                    f"{"[/link]\n" if self.remote_location.startswith('http') else "\n" }"
```